Assets/Scripts/Audio/FMODRadio.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FMODRadio : MonoBehaviour
+{
+    [FMODUnity.EventRef]
+    public string Music;
+    FMOD.Studio.EventInstance musicEv;
+    public bool isPlaying;
+
+    private Interactable interactable;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        interactable = Utils.GetRequiredComponent<Interactable>(this);
+        interactable.OnInteractEnd += ChangeStation;
+        musicEv = FMODUnity.RuntimeManager.CreateInstance(Music);
+        musicEv.set3DAttributes(FMODUnity.RuntimeUtils.To3DAttributes(transform.position)); 
+        musicEv.start();
+    }
+
+    public void ChangeStation(Interactable source)
+    {
+        if (isPlaying)
+        {
+            musicEv.stop(FMOD.Studio.STOP_MODE.IMMEDIATE);
+            isPlaying = false;
+        }
+
+        if (!isPlaying)
+        {
+            musicEv.start();
+            isPlaying = true;
+        }
```

Comment:
I just realized we can make this much simpler if you just want to change the station
```suggestion
        if (isPlaying)
        {
            musicEv.stop(FMOD.Studio.STOP_MODE.IMMEDIATE);
            musicEv.start();
        }
```

Reply:
made the edit and works perfectly!